routes/index.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable global-require */
+module.exports = {
+  usersRoutes: require('./usersRoutes')
```

Comment:
I think I prefer how files are loading in `controllers/index.js` then you can remove the disabled eslint rule.